person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Kindly add `:rentals` to the list of `attr_accessor` so that it can be accessed and set from outside this class.

  ```
  attr_accessor :name, :age, :rentals
  ```

Reply:
Sure thing @julie-ify! I forgot to add rentals in the list of `attr_accessor`. Thank you so much for pointing that out 💯 👍